src/alignment/nanopolish_eventalign.cpp

```diff
@@ -528,7 +540,16 @@ void realign_read(const ReadDB& read_db,
     std::string read_name = bam_get_qname(record);
 
     // load read
-    SquiggleRead sr(read_name, read_db, opt::write_samples ? SRF_LOAD_RAW_SAMPLES : 0);
+    int sr_flag;
+    if( opt::write_samples) {
+        sr_flag = 2;
```

Comment:
it is better to use `SRF_LOAD_RAW_SAMPLES` rather than the literal value here. Could also combine this with the `else if` below